app/views/spree/admin/variants/index.html.haml

```diff
@@ -27,9 +27,10 @@
         %td.align-center= variant.display_price.to_html
         %td.align-center= variant.sku
         %td.actions
-          = link_to_edit(variant, no_text: true) unless variant.deleted?
-          = link_to_delete(variant, no_text: true) unless variant.deleted?
+          = link_to_with_icon('icon-edit', Spree.t(:edit), edit_object_url(variant, @url_filters), no_text: true) unless variant.deleted?
+          = link_to_delete(variant, { url: object_url(variant, @url_filters), no_text: true }) unless variant.deleted?
 
+// TODO this
```

Comment:
Leftover?

Reply:
I think this needs to block the PR. Otherwise we will need to have commits after the test happens. We need to remove this before we test, ok?